content/applications/websites/livechat/chatbots.rst

```diff
@@ -111,7 +112,7 @@ Phone
 *****
 
 Similar to email, this step type prompts the visitor to enter their phone number, which can be used
-at a later time to follow up with additional information, or to schedule demos, and more.
+to follow up with additional information, schedule demos, and more.
 
 .. warning::
    Due to the vast number of formats used for phone numbers worldwide, responses to this step type
```

Comment:
Does this only allow for numbers and special characters, like a +? If so, that could be mentioned.

Reply:
reponses are not validated, and can include special characters, numbers, and letters. I added a note to the warning to specify this.